src/swarm-topography/Swarm/Game/Scenario/Topography/Structure/Recognition/Static.hs

```diff
@@ -48,19 +42,3 @@ data LocatedStructure = LocatedStructure
 instance HasLocation LocatedStructure where
   modifyLoc f (LocatedStructure x originalLoc) =
     LocatedStructure x $ f originalLoc
-
-data StaticStructureInfo b = StaticStructureInfo
-  { _structureDefs :: [SymmetryAnnotatedGrid (Maybe b)]
-  , _staticPlacements :: Map SubworldName [LocatedStructure]
-  }
-  deriving (Show)
```

Comment:
This `StaticStructureInfo` record was relocated to the `Swarm.Game.Scenario.Topography.Structure.Recognition.Type` module.  And its `_structureDefs` member is made redundant.